config/routes.rb

```diff
@@ -14,6 +14,9 @@
     post '/logout' => 'devise/sessions#destroy'
   end
 
+  resources :list do
```

Comment:
as mentioned previously, `resources :list` will accomplish the same thing as `resources :list do end` - you should remove the `do end`